src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardUserInfoDirectives.scala

```diff
@@ -26,11 +26,28 @@ trait StandardUserInfoDirectives extends UserInfoDirectives with LazyLogging wit
 
   def requireActiveUser(samRequestContext: SamRequestContext): Directive1[SamUser] = requireOidcHeaders.flatMap { oidcHeaders =>
     onSuccess {
-      getUserInfo(directoryDAO, registrationDAO, oidcHeaders, samRequestContext).unsafeToFuture()
+      val requireActiveUserIO = for {
+        user <- getSamUser(directoryDAO, registrationDAO, oidcHeaders, samRequestContext)
```

Comment:
This will generate 2 DB queries for every `/api` call, right? Was wondering if we could do it in 1 query like:
```
select * from sam_user u where exists (select * from sam_group where name = 'tos' and user_id = u.id)
```
Just an idea, I don't feel too strongly (the queries are probably quick).

Reply:
[see this prior comment](https://github.com/broadinstitute/sam/pull/722#discussion_r868471261)

I am also not super thrilled that ToS is implemented as a group and not a first class thing. Including ToS in the user lookup query would mean custom code paths to load users by pet, google id or azure id. And these would need to take in the name of the ToS group. It would probably be better if accepted ToS version were just a column on the sam_user table.

Reply:
addressed in https://github.com/broadinstitute/sam/pull/723